GVFS/GVFS.Notifications/VFSForGit.Mac/org.vfsforgit.usernotification.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>EnvironmentVariables</key>
+        <dict>
+            <!--
+            Although VFSForGit.app gets launched in the GUI context of actively logged-in user, it
+            does not seem to inherit any user-specific environment variables (like user customized
+            $PATH specified in shell ~/.profile). VFSForGit needs /usr/local/bin to be in its 
+            search PATH environment, so specifying it here, along with rest of the default paths 
+            that are set by launchd.
+            -->
+            <key>PATH</key>
+            <string>/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin</string>
```

Comment:
Why are we needing the user's environment with /usr/local/bin in the $PATH for the GUI app? We probably shouldn't be relying on their PATH for lauching things? Why couldn't we just use the absolute paths to executables that we need (e.g., /usr/local/bin/vfsforgit ... /gvfs )

Reply:
In our processes we can use the absolute path names, no problem. But other tools like `git` (which we will use in the upcoming PR to get git version string) seems to need `PATH` environment setting. 

I will double check this (and remove this if not needed) in the upcoming PR when I start using `git` and also make the app an actual launchd agent.